.editorconfig

```diff
@@ -0,0 +1,18 @@
+# EditorConfig configuration for trezor-user-env
+# https://EditorConfig.org
+
+# Top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file, utf-8 charset
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
+
+# Ignore diffs/patches
+[*.{diff,patch}]
+end_of_line = unset
+insert_final_newline = unset
+trim_trailing_whitespace = unset
```

Comment:
Yes, comically you have forgotten to add a newline at the end of your configuration file which enforces newlines :laughing:

Reply:
yep, because when i did it it removed the file and didn't fix it 🤦  Pushed update